generators/generator-bot-adaptive/generators/app/index.js

```diff
@@ -200,7 +204,7 @@ module.exports = class extends BaseGenerator {
             appSettings.runtime.command = `dotnet run --project ${botName}.csproj`;
         }
         break;
-      case platforms.js:
+      case platforms.js.name:
```

Comment:
Ignore my comment if you've already done this, but I would triple check that no other generators reference the `platforms` structure.